lib/clearance.rb

```diff
@@ -10,7 +10,4 @@
 require 'clearance/constraints'
 
 module Clearance
```

Comment:
Oh, probably. Then I'm evenly split between "Just remove this and add it when you start adding RDoc" and "Might as well leave it in."